app/models/assignment.rb

```diff
@@ -629,6 +629,7 @@ def summary_json(user)
       criteriaColumns: criteria_columns,
       numAssigned: self.get_num_assigned(user.instructor? ? nil : user.id),
       numMarked: self.get_num_marked(user.instructor? ? nil : user.id),
+      enableTest: self.assignment_properties&.enable_test || false,
```

Comment:
Oh sorry I didn't catch this earlier, but you can just use `self.enable_test`. If you're curious you can learn about the `delegate_missing_to` method.